### DISCUSSION TITLE
Guardian: Remove references to localhost

### DISCUSSION THREAD

**Discussion by voycey** [hiero-ledger/hiero-improvement-proposals#288] [Ideas]
As part of our steps to move the Guardian into production we need to remove all references of localhost and instead point these to the container names.

Services like Fargate and GKE do not easily support localhost + port as a naming convention but instead rely on container names to communicate.

This should be a simple change but I wanted to discuss based on planned deployment targets for the wider community and any gotchas that might exist there.

**Comment by anvabr** [comment]
SImilarly to https://github.com/hashgraph/hedera-improvement-proposal/discussions/287, @voycey, thank you for the review and suggestion above. The team behind Guardian will take it onboard. However this is not the right place to submit bug reports and enhancement/feature requests for Guardian. The best way is to create an issue in the guardian repo https://github.com/hashgraph/guardian. Just click on 'Issues' and 'New Issue' and write-up your notes and Guardian product team will process it accordingly.

**Comment by anvabr** [comment]
Recorded in: https://github.com/hashgraph/guardian/issues/217
